src/csharp/Grpc.IntegrationTesting.StressClient/Program.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
```

Comment:
Copyright/licence info here. (Looks like a the copyright sanity test failed due to this)